metadata-db-app/src/cmr/metadata_db/services/concept_service.clj

```diff
@@ -654,11 +660,13 @@
 
 (defmethod force-delete-cascading-events :service
   [context concept-type concept-id revision-id]
-  ;; delete the related service associations
-  (delete-associations context concept-type concept-id revision-id :service-association)
-  ;; Do not queue the concept revision deletion event for now.
-  ; (ingest-events/publish-concept-revision-delete-msg context concept-id revision-id)
-  )
+  (when (latest-revision? context concept-id revision-id)
```

Comment:
I think the check should be when this concept-id no longer exists or the latest revision is a tombstone delete the associations.

Reply:
This has to do with the conversation that @ygliuvt and I had yesterday afternoon that resulted in a bit of a direction change for this ticket; we talked again this morning, and are continuing to refine the approach. It might make sense for all of us to confer about this together at some point today ...

Reply:
What was the conclusion on this?